loopout.go

```diff
@@ -647,11 +640,13 @@ func (s *loopOutSwap) payInvoiceAsync(ctx context.Context,
 	}
 
 	maxRetries := 1
+	paymentTimeout := s.executeConfig.totalPaymentTimout
```

Comment:
Total payment timeout is now maximized in 1 hour (30 minutes by default).

Reply:
> is our htlc deadline measured in timestamp or in blocks

Timestamp, I believe the reasoning is that people in LN world think "I need inbound liquidity in 5 / 20 / 30 mins" rather than in blocks.